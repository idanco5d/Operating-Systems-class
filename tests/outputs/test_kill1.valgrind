==8745== Memcheck, a memory error detector
==8745== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==8745== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==8745== Command: /mnt/c/Users/illus/os/hw/1/smash
==8745== Parent PID: 8740
==8745== 
==8745== 
==8745== FILE DESCRIPTORS: 4 open (3 std) at exit.
==8745== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_kill1.valgrind
==8745==    <inherited from parent>
==8745== 
==8745== 
==8745== HEAP SUMMARY:
==8745==     in use at exit: 0 bytes in 0 blocks
==8745==   total heap usage: 60 allocs, 60 frees, 80,677 bytes allocated
==8745== 
==8745== All heap blocks were freed -- no leaks are possible
==8745== 
==8745== For lists of detected and suppressed errors, rerun with: -s
==8745== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
