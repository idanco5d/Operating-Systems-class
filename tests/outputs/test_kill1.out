smash> smash> smash> signal number 1 was sent to pid 9328
smash> received signal number: 1
smash> smash: sending SIGKILL signal to 1 jobs:
9328: ./printSignals.exe &
received signal number: 15
